src/components/reporter/Form.js

```diff
@@ -3,79 +3,82 @@ import { StaticQuery, graphql } from "gatsby";
 import clsx from "clsx";
 import Agreement from "../content/Agreement.js";
 import * as styles from "./Form.module.scss";
-import { Widgets } from "./widgets";
+import LinesOfCode from "../reporter/LinesOfCodeInput.js";
+import * as widgetStyles from "./widgets/Widgets.module.scss";
+import { Widget } from "./widgets";
+import FormField from "./widgets/FormField";
 
 const config = {
   labelAll: "bug",
-  fields: [
-    {
-      name: "handle",
-      label: "Handle",
-      helpText: "Handle you're competing under (individual or team name)",
-      widget: "warden",
-      required: true,
-      options: [],
-    },
-    {
-      name: "email",
-      label: "Email address",
-      helpText: "Used to send a copy of this form for your records",
-      widget: "text",
-      required: true,
-    },
+};
+
+const emailField = {
+  name: "email",
+  label: "Email address",
+  helpText: "Used to send a copy of this form for your records",
+  widget: "text",
+  required: true,
+};
+
+const addressField = {
+  name: "address",
+  label: "Polygon address",
+  helpText:
+    "Address where your prize should go to (and retroactive token reward should C4 be tokenized later). If you use a smart contract wallet, please contact one of our organizers in Discord in addition to adding the address here.",
```

Comment:
`(and retroactive token reward should C4 be tokenized later)` isn't necessary anymore :)